src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,47 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+const centuries = ['16', '17', '18', '19', '20'];
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuriesUrl = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const handleChangeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const search = getSearchWith(
+      searchParams, { query: event.target.value || null },
+    );
+
+    setSearchParams(search);
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          className={cn({ 'is-active': !searchParams.get('sex') })}
+          params={{ sex: null }}
+        >
+          All
```

Comment:
Not critical, but it's better to use Enums\variables instead of the hardcoded values